tests/integration/cartography/intel/aws/test_rds_event_subscriptions.py

```diff
@@ -0,0 +1,247 @@
+    _ensure_local_neo4j_has_test_account(neo4j_session)
+    _ensure_local_neo4j_has_test_sns_topics(neo4j_session)
+    _ensure_local_neo4j_has_test_rds_resources(neo4j_session)
+    _ensure_local_neo4j_has_test_event_subscriptions(neo4j_session)
+
+    # Test RESOURCE relationship to AWSAccount
+    expected_account = {
+        (TEST_ACCOUNT_ID, s["CustSubscriptionId"])
+        for s in DESCRIBE_EVENT_SUBSCRIPTIONS_RESPONSE["EventSubscriptionsList"]
+    }
+    actual_account = check_rels(
+        neo4j_session,
+        "AWSAccount",
+        "id",
+        "RDSEventSubscription",
+        "id",
+        "RESOURCE",
+        rel_direction_right=True,
+    )
+    assert actual_account == expected_account
+
+    # Test NOTIFIES relationship to SNSTopic
+    expected_sns = {
+        (s["CustSubscriptionId"], s["SnsTopicArn"])
+        for s in DESCRIBE_EVENT_SUBSCRIPTIONS_RESPONSE["EventSubscriptionsList"]
+    }
+    actual_sns = check_rels(
+        neo4j_session,
+        "RDSEventSubscription",
+        "id",
+        "SNSTopic",
+        "arn",
+        "NOTIFIES",
+        rel_direction_right=True,
+    )
+    assert actual_sns == expected_sns
+
+    # Test MONITORS relationship to RDSInstance
+    expected_instances = {
+        (s["CustSubscriptionId"], id)
+        for s in DESCRIBE_EVENT_SUBSCRIPTIONS_RESPONSE["EventSubscriptionsList"]
+        if s["SourceType"] == "db-instance"
+        for id in s["SourceIdsList"]
+    }
+    actual_instances = check_rels(
+        neo4j_session,
+        "RDSEventSubscription",
+        "id",
+        "RDSInstance",
+        "db_instance_identifier",
+        "MONITORS",
+        rel_direction_right=True,
+    )
+    assert actual_instances == expected_instances
+
+    # Test MONITORS relationship to RDSCluster
+    expected_clusters = {
+        (s["CustSubscriptionId"], id)
+        for s in DESCRIBE_EVENT_SUBSCRIPTIONS_RESPONSE["EventSubscriptionsList"]
+        if s["SourceType"] == "db-cluster"
+        for id in s["SourceIdsList"]
+    }
+    actual_clusters = check_rels(
+        neo4j_session,
+        "RDSEventSubscription",
+        "id",
+        "RDSCluster",
+        "db_cluster_identifier",
+        "MONITORS",
+        rel_direction_right=True,
+    )
+    assert actual_clusters == expected_clusters
+
+    # Test MONITORS relationship to RDSSnapshot
+    expected_snapshots = {
+        (s["CustSubscriptionId"], id)
+        for s in DESCRIBE_EVENT_SUBSCRIPTIONS_RESPONSE["EventSubscriptionsList"]
+        if s["SourceType"] == "db-snapshot"
+        for id in s["SourceIdsList"]
+    }
+    actual_snapshots = check_rels(
+        neo4j_session,
+        "RDSEventSubscription",
+        "id",
+        "RDSSnapshot",
+        "db_snapshot_identifier",
+        "MONITORS",
+        rel_direction_right=True,
+    )
+    assert actual_snapshots == expected_snapshots
+
+
+def test_cleanup_event_subscriptions(neo4j_session):
```

Comment:
Can remove this test. If we test for the sync function it will be basically covered anyway. We have other for the core framework that ensure the cleanups handle as expected too.